README.md

```diff
@@ -288,10 +288,14 @@ To start a new session in the CloudControl context, run `createSession <token>`
 
 #### Configuration
 
-* Environment AWS_ACCESS_KEY_ID: Specifies an AWS access key associated with an IAM user or role
-* Environment AWS_SECRET_ACCESS_KEY: Specifies the secret key associated with the access key. This is essentially the password for the access key
-* Environment AWS_DEFAULT_REGION: Specifies the AWS Region to send the request to
-* Environment AWS_MFA_ARN: ARN of the MFA device to use to log in
+* 
```

Comment:
What is the purpose of putting an asterix on a single line?

Reply:
None. I thought I have no choice but to do this, but actually found another way. I've pushed an update to this.